blog/_posts/2020-10-04-vsm-and-kaizen-modules-added.md

```diff
@@ -0,0 +1,26 @@
+Following the positive community reaction to the launch of the Online DevOps Dojo, we are pleased to announce we have added two more modules to the dojo
+
+* Value Stream Mapping
+* DevOps Kaizen
+
+These are in addition to the 5 modules which were included in the initial launch:
+
+* Welcome and Setup
+* Leading Change
+* Version Control
+* Continuous Integration
+* Shift Security Left
+
+## VSM and DevOps Kaizen Modules
+
+Value Stream Mapping is a technique we encourage teams to understand the current state and to document the desired state of their process aka their value stream.
```

Comment:
The initial wording seems a bit off.  Is the intent a) we encourage teams to use the technique and b) the technique does XYZ?  if so may need a bit more to it, but happy to collaborate on it - can suggest something if it helps

Reply:
Done